GettingToKnowMe.php

```diff
@@ -0,0 +1,61 @@
+<html>
+  <head>
+    <title> Getting to know me </title>
+    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
+  </head>
+<body>
+    <h1>
+        My Blog! </h1>
+        <div class = "active">
+              <ul>
+              <li>
+                  <a href = "index.php">
+                  Home
```

Comment:
Here's another place where the indentations are a bit weird. One thing I notice is that you're using spaces rather than tabs (i.e. you're entering ~4 spaces instead of one "tab" character which displays as four spaces). There's an age-old debate about whether tabs or spaces are better, so I won't tell you what to do, but our convention with our codebase at LACRM is to use tabs. If you're using spaces because you prefer them, no problem. If you didn't even realize you were using spaces and you don't have a preference, I'd encourage you to try tabs as I think it makes it easier to keep your page indented correctly. Let me know if you'd like to discuss this more.